Models/Products.cs

```diff
@@ -13,8 +14,8 @@ public class Products
         
 
         [Required]
```

Comment:
needs to be deleted

Reply:
Brett is working on this deletion from Products.